### DISCUSSION TITLE
Target journals

### DISCUSSION THREAD

**Discussion by kyle-messier** [NIEHS/chopin#31] [General]
@sigmafelix  As you begin to pull together this package, we need to the about impactful journals. 
Computational statistics, GIS, Big Data, are common themes. Correct me if I am wrong, but we [you] are not developing any novel methods, correct? An efficient algorithm would count as a novel method. If so, that means novel methodology cannot be a requirement of the journal. 

Some journals to think about:

1. IEEE Transactions of Big Data (IF = 7.2)
2. Journal of Machine Learning Research (Machine Learning Open Source Software)- There is a special issue on climate change (https://www.jmlr.org/special_issues/climate_change.html) 
3. Journal of Open Source Software

**Comment by kyle-messier** [comment]
Adding 4. Journal of Agricultural, Biological, and Environmental Statistics (JABES). I'll ask my colleague, the former EIC, if he thinks it is appropriate.

**Comment by sigmafelix** [comment]
Thank you for the suggestion. I think 5. GeoHealth could also be a suitable journal.

**Comment by sigmafelix** [comment]
6. Environmental Modelling & Software
